varnish/src/metrics_reader.rs

```diff
@@ -223,46 +230,48 @@ pub enum Format {
     Bitmap,
     /// Time
     Duration,
-    /// Unit unknown
```

Comment:
As mentioned, I'm very much against that change, `vmods` can create unknown types, and we can't fail on those...

Reply:
If `unknown` is a valid value, we certainly can keep it - but in that case it cannot be `Unknown`, but rather `Custom(c_int)` or something similar, i.e. it must represent some state, rather than be an uncertain value.

Reply:
I agree, do you want to do it in this PR, or do that later?

Reply:
separate PR

Reply:
Created https://github.com/varnish-rs/varnish-rs/issues/178 to remember